hdf/src/hdatainfo.h

```diff
@@ -11,7 +11,7 @@
  * help@hdfgroup.org.                                                        *
  * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 
-/* $Id: hproto.h 5400 2010-04-22 03:45:32Z bmribler $ */
+/* $Id: hdatainfo.h 5400 2010-04-22 03:45:32Z bmribler $ */
```

Comment:
Is this line necessary?

Reply:
No. These are an artifact of CVS.

Reply:
I know, the other PR took care of that, so I just left it there for github to do its thing.  I can remove now if you think I should.

Reply:
Allen, do you receive this notification?